src/lib.rs

```diff
@@ -107,6 +108,7 @@
 //! 10. [`#[derive(AddAssign)]`](add_assign.html)
 //! 11. [`#[derive(MulAssign)]`](mul_assign.html)
 //! 12. [`#[derive(Constructor)]`](constructor.html)
+//! 13. [`#[derive(TryInto)]`](try_info.html)
```

Comment:
should be `try_into.html`

Reply:
and can you place this Below `FromStr` in this list? I like to keep the same ordering of traits throughout the docs.